gui.js

```diff
@@ -190,6 +190,14 @@ define(function(require, module, exports) {
                             }
 
                             var c = 300;
+                             // START - logic to sort custom Builders
+                            if(names.length > 5) {
```

Comment:
this won't work since different workspace types can have different number of builders, would it be ok to sort all of them or do you need custom and user defined builders to be separate?